react/react2/week1/homework/src/App.js

```diff
@@ -23,28 +26,31 @@ function App() {
   useEffect(() => {
     fetchAPI()
   }, [fetchAPI])
+  const title = "Github user searcher"
+  const AppValues = [title]
   return (
     <div className="App">
-      <h1> Github user searcher </h1>
+      {/* All state should be managed using the context api! */}
+      <AppContext.Provider value={AppValues}>
+      <H1Title /> 
       {/* When the search is loading, show "loading..." */}
-      {loading ? "Loading" : 
+      {loading ? "loading..." : 
         <>
-          <input value={user} onChange={inputHandler} placeholder="Search for user" />
+          <input value={user} onChange={inputHandler} placeholder="Search for user" /><br />
           {/* When the search result is empty (no users), you should show "No results..." */}
           {data.total_count === 0 ? "No results..." :
             <>
               {data.items.map(item => {
-                return item.login + item.url
+                return <><a target="_blank" rel="noreferrer" href={item.url}>{item.login}</a><br /></>
```

Comment:
I would suggest using block elements, then <br/> could be avoided, which should be avoided as much as possible because it comes from HTML, it's not semantic, hence not so useful for dynamic content